frontend-react/e2e/mocks/receiverStatus.ts

```diff
@@ -0,0 +1,172 @@
+import {
+    addMinutes,
+    addSeconds,
+    differenceInMinutes,
+    endOfDay,
+    interval,
+    startOfDay,
+    subDays,
+} from "date-fns";
+import { randomInt } from "crypto";
+import { RSReceiverStatus } from "../../src/hooks/api/UseReceiversConnectionStatus/UseReceiversConnectionStatus";
+
+const mockFailResult = "ERROR";
+const mockSuccessResult = "SUCCESS";
+
+/**
+ * Create a set of mock statuses for a specific org's receiver. Default day range
+ * is 2 days ago to today (UTC) if one is not provided. Statuses will
+ * be created once per `maxMinutesPerStatus` minutes (default is 2 hours aka
+ * 120 minutes).
+ */
+export function createMockGetReceiverStatusSet({
+    range,
+    maxMinutesPerStatus = 2 * 60,
+    receiver,
+    statusType = "mix",
+    randomlySkip,
+    statusIdStart = 1,
+}: {
+    range?: [start: Date, end: Date];
+    maxMinutesPerStatus?: number;
+    statusType?: "success" | "fail" | "mix";
```

Comment:
is there a pre-existing `enum` somewhere we can draw this from?

Reply:
`SuccessRate` enum now used